.circleci/config.yml

```diff
@@ -2145,10 +2145,16 @@ jobs:
       - <<: *STEP_CHECKOUT
       - <<: *STEP_ATTACH_WORKSPACE
       - run:
+          # TODO: removes this step once host-in-runner is merged on system tests
           name: Install good version of docker-compose
           command: |
             sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
             sudo chmod +x /usr/local/bin/docker-compose
+
+      - run:
+          name: Install python 3.9
+          command: sudo apt-get install python3.9-venv
```

Comment:
I recommend doing an apt update within the command to prevent issues when the image has stale apt cache.